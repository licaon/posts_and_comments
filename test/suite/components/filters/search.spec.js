import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import Search from 'src/components/filters/search';

describe('Search component', () => {
  const createComponent = (props) => {
    const wrapper = shallow(
      <Search
        {...props}
      />);
    return toJson(wrapper);
  };
  it('should render component correctly', (done) => {
    const component = createComponent({
      value: '1',
      onValueChange: () => {},
    });

    expect(component).toMatchSnapshot();
    done();
  });
});
