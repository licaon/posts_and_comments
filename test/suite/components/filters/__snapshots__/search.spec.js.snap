// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search component should render component correctly 1`] = `
<TextField
  defaultValue="1"
  fullWidth={true}
  id="search"
  label="Search"
  leftIconStateful={true}
  lineDirection="center"
  onChange={[Function]}
  passwordIconChildren="remove_red_eye"
  rightIconStateful={true}
  type="text"
/>
`;
